extensions/avro/src/test/java/com/hazelcast/jet/avro/AvroSinkTest.java

```diff
@@ -47,6 +50,7 @@
 import static org.junit.Assert.assertTrue;
 
 @RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelJVMTest.class})
```

Comment:
No, I have not considered it, since it's a concern independent from what we are doing here. Or at least I think so; we are basically marking the tests as "parallelly runnable", how the build chooses to do that should be a separate concern. Is it not?

Reply:
Agree with Jozsef, we'll burn that bridge once we get there :) But fair point.